utils/backup.go

```diff
@@ -59,6 +60,15 @@ func Restore(backupFile string, databaseFile string) error {
 func Backup(db *sql.DB, backupFile string) {
 	log.Traceln("Backing up database to", backupFile)
 
+  BackupDirectory := filepath.Dir(backupFile)
+  _, err := os.Stat(BackupDirectory)
```

Comment:
There's a little utility function you can use so you don't have to do the extra Stat.
https://github.com/owncast/owncast/blob/develop/utils/utils.go#L29

Reply:
I used the DoesFileExists() and also changed the permission on the created folder to 700.